run.py

```diff
@@ -83,11 +83,6 @@ def prepare():
     #    with cd(root):
     #        subprocess.call((sys.executable, "setup.py", "build"), shell=False)
 
-    # Notify the help menu that the Sphinx documentation is in a different
-    # place than it otherwise would be.
-    docpath = joinpath(root, 'docs', 'sphinx-docs', '_build', 'html')
-    os.environ['SASVIEW_DOC_PATH'] = docpath
-
```

Comment:
This might be related to the issue noted by @jamescrake-merani. `run.py` does not require a developer to locally build, so the `build/docs` directory may not exist.

Reply:
I couldn't find any code that uses `SASVIEW_DOC_PATH` in sasview - hence removing it in this PR. The last mention of it in git (other than run.py) I can find is the removal of the wx sasgui (83b31818f).

I suspect that the answer is that in a fresh checkout, nothing has tried to build the docs at all when the user goes straight to `run.py`. Running the build machinery without build the wheel should be enough: `hatchling build --hooks-only`; I'll do some experimenting on this as part of thinking about the documentation updates that are needed.